packages/web/src/common/store/cache/sagas.js

```diff
@@ -327,27 +282,45 @@ function* watchRemove() {
 
 function* initializeCacheType() {
   const remoteConfig = yield getContext('remoteConfigInstance')
+  const getFeatureEnabled = yield getContext('getFeatureEnabled')
+  const isNativeMobile = yield getContext('isNativeMobile')
   yield call(remoteConfig.waitForRemoteConfig)
-  const fastCache = yield call(
-    remoteConfig.getFeatureEnabled,
-    FeatureFlags.FAST_CACHE
+
+  const fastCache = yield call(getFeatureEnabled, FeatureFlags.FAST_CACHE)
+  const fastMobileCache = yield call(
+    getFeatureEnabled,
+    FeatureFlags.FAST_MOBILE_CACHE
   )
   const safeFastCache = yield call(
-    remoteConfig.getFeatureEnabled,
+    getFeatureEnabled,
     FeatureFlags.SAFE_FAST_CACHE
   )
 
-  if (fastCache) {
-    yield put(cacheActions.setCacheType({ cacheType: 'fast' }))
+  let cacheType = 'normal'
+
+  if (fastCache || (isNativeMobile && fastMobileCache)) {
```

Comment:
do we want to be able to enable it for web but not mobile? seems like we can either do both or just mobile this way

Reply:
thanks for the callout here, i realized we dont need a separate "fast" and fast-mobile" now that desktop issues are fixed. this was to allow us to roll forward with the fast cache on mobile cause desktop was acting up for a while there. but we are consistent now, so will remove.